tests/build/build_templates.go

```diff
@@ -20,6 +22,12 @@ import (
 	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	ecPipelineRunTimeout     = time.Duration(10 * time.Minute)
+	chainsAttestationTimeout = time.Duration(60 * time.Minute)
```

Comment:
1h seems really long, it worked with 10min [before](https://github.com/redhat-appstudio/e2e-tests/pull/743/files#diff-420007369ea2e1a8133dc8c9a8fc032202afa2a4fc08e7f8753701ae4ab87333L261), could it be enough now?

```suggestion
	chainsAttestationTimeout = time.Duration(10 * time.Minute)
```